src/main/java/com/amihaiemil/docker/RtVersion.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2018-2019, Mihai Emil Andronache
+ * All rights reserved.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 1)Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ * 2)Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * 3)Neither the name of docker-java-api nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.amihaiemil.docker;
+
+import org.apache.http.client.HttpClient;
+
+import javax.json.JsonObject;
+import java.net.URI;
+
+/**
+ * Runtime {@link Version}.
+ * @author Michael Lux (michi.lux@gmail.com)
+ * @since 0.0.11
+ */
+final class RtVersion extends JsonResource implements Version {
+    /**
+     * Apache HttpClient which sends the requests.
+     */
+    private final HttpClient client;
+
+    /**
+     * Base URI.
+     */
+    private final URI baseUri;
+
+    /**
+     * Docker API.
+     */
+    private final Docker docker;
+
+    /**
+     * Ctor.
+     * @param rep JsonObject representation of this Version.
+     * @param client The http client.
+     * @param uri The URI for this version.
+     * @param dkr The docker entry point.
+     * @checkstyle ParameterNumber (5 lines)
+     */
+    RtVersion(
+        final JsonObject rep, final HttpClient client,
+        final URI uri, final Docker dkr
+    ) {
+        super(rep);
+        this.client = client;
+        this.baseUri = uri;
+        this.docker = dkr;
+    }
+}
```

Comment:
@milux We also need unit tests. See how other unit tests are written, it's not complicated. Generally, PRs should not decrease code coverage.

Reply:
Done, although the method I wrote the unit test is somewhat different from that of the other tests. (Using locally stored JSON resource file for response mockup, for instance.) Hope you're still fine with this...

Reply:
@milux Yes, it's fine. It can be changed later, no prob.